src/app/views/results.html

```diff
@@ -1,10 +1,27 @@
-<div class="consigna__container">
-  <h2>ACA TIENEN QUE CARGAR UNA GRILLA CON LOS ARTISTAS QUE APARECIERON COMO RESULTADO DE LA BUSQUEDA ANTERIOR</h2>
-  <img src="https://i.scdn.co/image/445ffa41207f321883fb01f406165801c7f41ef0" alt="">
-  <h4>Cyndi Lauper</h4>
-  <img src="https://i.scdn.co/image/6b86adfb6637e08f76390266b617120e95ad16c0" alt="">
-  <h4>Cindy Bradley</h4>
-  <img src="https://i.scdn.co/image/82633230315fb6b7cc7f4b573bc9e8edf0da69da" alt="">
-  <h4>Cindy Bullens</h4>
-  
+<div ng-controller="resultsCtrl">
+    <nav class="nav">
+        <img src="https://lh3.googleusercontent.com/UrY7BAZ-XfXGpfkeWg0zCCeo-7ras4DCoRalC_WXXWTK9q5b0Iw7B0YQMsVxZaNB7DM=w300" alt="" class="logo-center">
+    </nav>
+    <div class="inside-container">
+        <div class="row">
+            <h1 class="title">Artists</h1>
+            <p>You are currently searching: {{resultsCtrl.artist}}</p>
+        </div>
+        <div class="row">
+            <input type="text" class="search-input" placeholder="Type the name of your favourite artist" ng-model="resultsCtrl.artist">
+            <a href=""><img src="http://www.freeiconspng.com/uploads/search-icon-png-17.png" id="search-icon" ng-click="resultsCtrl.getArtists()"></a>
```

Comment:
Use font-awesome instead of image icons, is more lightweight and you can stylish icons with css (color, size, etc).